client/components/common/tce-core/QuestionContainer/index.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card class="tce-question-container my-2 grey lighten-5">
+  <v-card class="tce-question-container my-2 grey lighten-5 elevation-0">
```

Comment:
This component is reused / you removed the elevation within the standard container aswell.